azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/KnownLinuxVirtualMachineImage.java

```diff
@@ -17,10 +17,16 @@
     DEBIAN_8("credativ", "Debian", "8"),
     /** CentOS 7.2. */
     CENTOS_7_2("OpenLogic", "CentOS", "7.2"),
-    /** OpenSUSE-Leap 42.1. */
+    /**
+     * OpenSUSE-Leap 42.1.
+     * @deprecated use withLatestLinuxImage("SUSE", "openSUSE-Leap", "42.3") instead.
```

Comment:
when I used the method name without `#` in the Javadoc, i got a warning in Eclipse. Does javadocs still resolve this reference properly when no `#` is used? (in other words: `#withLatestLinuxImage(...)`)

Reply:
thanks!, updated PR so that these references resolve properly.